base/stat.jl

```diff
@@ -53,8 +53,8 @@ StatStruct(buf::Union{Vector{UInt8},Ptr{UInt8}}) = StatStruct(
     ccall(:jl_stat_size,    UInt64,  (Ptr{UInt8},), buf),
     ccall(:jl_stat_blksize, UInt64,  (Ptr{UInt8},), buf),
     ccall(:jl_stat_blocks,  UInt64,  (Ptr{UInt8},), buf),
-    ccall(:jl_stat_mtime,   Float64, (Ptr{UInt8},), buf),
-    ccall(:jl_stat_ctime,   Float64, (Ptr{UInt8},), buf),
+    ccall(:jl_stat_mtime,   Float64, (Ptr{UInt8},), buf) |> Dates.unix2datetime,
```

Comment:
Maybe it makes sense to define DateTime in libc? There's a bunch of time stuff there already (including the parallel TimeVal class) and libc is loaded before filesystem.jl